system_cvterms.txt

```diff
@@ -1,207 +1,207 @@
 cv name	cvterm name	definition	date	person	dbpatch #	comment
-calendar	Fertilizer Event				00052	
```

Comment:
this file is a hard tab separated file. atom's whitespace add-on is clearing the end tabs. i would just disable the whitespace add-on so that it does not do these kinds of edits

Reply:
good catch! I had no idea that was enabled by default, just disabled it. I'll checkout the copy from master and redo the edits.